### PR TITLE
Change any increment serving counter.

### PR DESCRIPTION
**Pull request by kght6123**
*Issue #, if available:*

None.

*Description of changes:*

Made it possible to change the increment value of the Serving Counter to any value. You can also specify a negative value to decrease the Serving Counter.

スクリーンショット 2022-12-24 23 26 33

### CODE REVIEW

File: source/control-panel/src/components/IncrementServingCounter.vue [hunk 1]
```diff
@@ -34,7 +34,7 @@ increment the serving counter of the waiting room -->
       </button>
       <!-- display the slider value -->
       <div class="m-2 mx-5">
-        Increment by: <span class="mx-2 h3">{{ increment }}</span>
+        Increment by: <input type="number" v-model="increment" class="form-control form-control-lg w-50" />
```
Comment (thyagram-aws) [inline source/control-panel/src/components/IncrementServingCounter.vue:37]: we should add a min to the input
`Increment by: <input type="number" v-model="increment" class="form-control form-control-lg w-50", min=1 />`

File: source/control-panel/src/components/IncrementServingCounter.vue [hunk 2]
```diff
@@ -19,7 +19,7 @@ increment the serving counter of the waiting room -->
       <!-- add a label and slider for the increment value -->
       <label class="form-label"></label>
       <!-- connect the slider to the data model and input handler -->
-      <input type="range" class="form-range" min="1" max="1000" step="1" v-model="increment" @input="newValue = true" />
+      <input type="range" class="form-range" min="-1000" max="1000" step="1" v-model="increment" @input="newValue = true" />
```
[the remark this answers is not in the document]
Comment (thyagram-aws) [reply source/control-panel/src/components/IncrementServingCounter.vue:22]: @kght6123 moving the serving counter behind will have undesired effects on queue position expiry controlled through the ENABLE_QUEUE_POSITION_EXPIRY flag.

**Review by kght6123 — COMMENTED** [review]
@thyagram-aws Fixed. Please check when you have time.

File: source/control-panel/src/components/IncrementServingCounter.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (kght6123) [reply source/control-panel/src/components/IncrementServingCounter.vue]: @thyagram-aws Fixed. Please check when you have time.
Disallowed input of negative values.

**Review by thyagram-aws — APPROVED** [review]

**Pull request merged**